src/Model/Category/Category.php

```diff
@@ -34,7 +34,7 @@
  *
  */
 
-namespace Nosto\Model;
+namespace Nosto\Model\Category;
```

Comment:
This introduces a BC break, make sure to bump the next release to a major version